tests/core/test_core.py

```diff
@@ -117,11 +117,14 @@ def test_show_all_messages(self, mocker, controller, index_all_messages):
         controller.model.muted_streams = []
         controller.model.muted_topics = []
         controller.show_all_messages('')
+
```

Comment:
I initially added these since there were tests which used this data in the past, I believe, and it was clearer with lines separating the collection of data for each assert, where they are more complex. However after rebasing to master these seem to not be used; we could remove them - I think the assert was with regard to how `msg_list.set_focus` was called.

Reply:
Yeah, they are most probably not used. In which case it would benefit to remove them altogether. :+1: